io_framework/csv_file_to_db.py

```diff
@@ -1,22 +1,24 @@
 from influxdb import InfluxDBClient
-from resources.config import RESOURCES_DIR
 import progressbar
 
-def csv_file_to_db(host, port, username, password, database, filename):
+
+def csv_file_to_db(host, port, username, password, database, filepath, measurement,
+                   label_to_use, field_name_to_use, drop_db):
```

Comment:
does label_to_use have a default value. You should add one

Reply:
also for drop_db

Reply:
Default values specified in the wrapper function in CsvWriter class.

Reply:
I see them. Cool.